sr_ur_arm_calibration_loader/src/sr_ur_arm_calibration_loader/sr_ur_arm_calibration_loader.py

```diff
@@ -96,28 +94,12 @@ def _get_serial_from_arm(self, arm_ip):
                           " Arm will NOT be calibrated. Ignore if running URSim.".format(ssh_exception_message))
         return arm_serial_number
 
-    def _check_arm_calibration_exists(self, arm_serial):
+    def _arm_calibration_exists(self, arm_serial):
         arm_calibration_file = os.path.join(self._arm_calibrations_folder, arm_serial + '.yaml')
         if os.path.isfile(arm_calibration_file):
```

Comment:
This is not part of the changes, but speaking of tidying, could we just return os.path.isfile(arm_calibration_file)?

Reply:
yeah that's much nicer, thanks :) 0cdf16ac084da1114813586548137bda29b55271